client/my-sites/google-my-business/index.js

```diff
@@ -69,6 +88,7 @@ export default function( router ) {
 		'/google-my-business/select-business-type/:site',
 		redirectLoggedOut,
 		siteSelection,
+		redirectToStatsIfLocationConnected,
```

Comment:
Same here, we should still be able to access this page even if we're connected I think